.circleci/config.yml

```diff
@@ -86,6 +86,32 @@ commands:
       - default_setup
       - run: << parameters.command >>
 
+  save_unit_test_results:
+    steps:
+      - run:
+          name: Store the tests result
+          command: |
+            mkdir -p ~/test-results/
+            mv application/target/surefire-reports/*.xml  ~/test-results/ ;
+      - store_test_results:
+          path: ~/test-results
+
+  save_spotbugs_results:
+    steps:
+      - run:
+          name: Store the spotbugs result
+          command:
+            find . -type f -regex ".*/target/spotbugsXml.xml" -exec cp {} ~/check-results/ \;
```

Comment:
You need to make sure every spotbugs report is saved. 
Spotbugs generates a ` spotbugsXml.xml ` file for each submodule and since you are storing them all in the same directory, withour renaming them, you will only store one ` spotbugsXml.xml ` file.

Reply:
Changes applied. Now, the reports are stored in different directories.